app/NotificationService.swift

```diff
@@ -1,4 +1,40 @@
 import BareKit
+import CallKit
+import OSLog
+import UserNotifications
+
+let logger = Logger(
```

Comment:
I did not catched `print` in the console, this is why I let it.

Reply:
They only show up when directly attaching to the extension.